packages/sdk/src/integration/server.ts

```diff
@@ -188,6 +188,17 @@ export const integrationHandler =
         const runtimeError = new RuntimeError(thrown.message, thrown)
         return { status: runtimeError.code, body: JSON.stringify(runtimeError.toJSON()) }
       }
+
+      if (thrown instanceof Error) {
```

Comment:
**chore:** you can remove this if statement; this is the behavior we want no matter the type of the thrown variable